test/CMakeLists.txt

```diff
@@ -325,6 +326,16 @@ itk_add_test(NAME itkInverse1DFFTImageFilterTest
     ${CMAKE_CURRENT_SOURCE_DIR}/Input/itkForward1DFFTImageFilterTestBaseline
     ${ITK_TEST_OUTPUT_DIR}/itkInverse1DFFTImageFilterTestOutput.mha
     )
+
+itk_add_test(NAME itkSpeckleReducingAnisotropicDiffusionImageFilterTest
+  COMMAND UltrasoundTestDriver
+  --compare
```

Comment:
```suggestion
  --compare
  --compareIntensityTolerance 0.0 
```

Reply:
I initially got an error but reversing the order of the parameters succeeded:
> --compareIntensityTolerance 0.01
  --compare

Leaving a small tolerance to safeguard against small internal changes / system differences.

Reply:
Yes, small tolerance should be kept. `0.0` is for sanity checking 😖 which you obviously wanted 😄.